src/factories/guide.ts

```diff
@@ -27,23 +30,30 @@ export async function guideFactory() {
   const label = inter('')
   element.addChild(label)
 
+  let scale = await waitForScale()
   let currentDate: Date | undefined
   let currentLabelFormatter: FormatDate
 
+  emitter.on('scaleUpdated', updated => scale = updated)
+
   application.ticker.add(() => {
+    if (settings.disableGuides) {
+      return
+    }
```

Comment:
Tracking if guides should be rendered or not externally same as the other things that can be disabled (animations, edges)